src/GLX/glvnd_genentry.c

```diff
@@ -166,7 +179,15 @@ static void *DefaultDispatchFunc(void)
 
 static GLVNDentrypointStub GetEntrypointStub(int index)
 {
+#if defined(__CHERI_PURE_CAPABILITY__)
+    const ptraddr_t sentry_addr = __builtin_cheri_address_get(glx_entrypoint_start);
+    const ptraddr_t stub_addr = sentry_addr + (index * STUB_SIZE);
+    const void* pcc = __builtin_cheri_program_counter_get();
```

Comment:
`void *pcc`

And do these really all need to be const? Sure, it's correct to, but the code style doesn't seem to do that from the context I can see. Especially for pcc, you've made it `const void *` not `void * const` (nor `const void * const`).